<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtractWaveformSettings</class>
 <widget class="QDialog" name="ExtractWaveformSettings">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="ref_groupBox">
     <property name="title">
      <string>Reference</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QRadioButton" name="channel_ref_radioButton">
          <property name="text">
           <string>Use single channel.</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="channel_ref_comboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QRadioButton" name="median_ref_radioButton">
          <property name="text">
           <string>Use median of channels.</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="show_channels_lineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="select_channels_toolButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="filter_groupBox">
     <property name="title">
      <string>Filter raw</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="filter_low_label">
          <property name="text">
           <string>Low cutoff</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="filter_low_doubleSpinBox">
          <property name="maximum">
           <double>30000.000000000000000</double>
          </property>
          <property name="value">
           <double>250.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="filter_high_label">
          <property name="text">
           <string>High cutoff</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="filter_high_doubleSpinBox">
          <property name="maximum">
           <double>30000.000000000000000</double>
          </property>
          <property name="value">
           <double>6000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="thr_groupBox">
     <property name="title">
      <string>Threshold</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="const_thr_radioButton">
          <property name="text">
           <string>Use constant threshold.</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="const_thr_doubleSpinBox">
          <property name="minimum">
           <double>-32768.000000000000000</double>
          </property>
          <property name="maximum">
           <double>32767.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="mad_thr_radioButton">
          <property name="text">
           <string>Use n times estimated SD of the noise.</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QDoubleSpinBox" name="mad_thr_doubleSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <double>-99.989999999999995</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="sigmas_label">
            <property name="text">
             <string>sigmas</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExtractWaveformSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExtractWaveformSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>channel_ref_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>channel_ref_comboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>median_ref_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>select_channels_toolButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>const_thr_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>const_thr_doubleSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mad_thr_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>mad_thr_doubleSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
